src/components/util/search-filters.js

```diff
@@ -33,11 +32,11 @@ export function getUnselectHrefIfSelectedFromResponseFilters(term, facet, filter
             toFilter    = _.findWhere(filters, { 'field' : field + '.to',   'term' : term.to }),
             fromFilter  = _.findWhere(filters, { 'field' : field + '.from', 'term' : term.from });
         } else if (facet.aggregation_type === 'date_histogram'){
-            var interval = getDateHistogramIntervalFromFacet(facet) || 'month',
-                toDate = moment.utc(term.key);
-            toDate.add(1, interval + 's');
-            toFilter    = _.findWhere(filters, { 'field' : field + '.to',   'term' : toDate.format().slice(0,10) }),
-            fromFilter  = _.findWhere(filters, { 'field' : field + '.from', 'term' : term.key });
+            // var interval = getDateHistogramIntervalFromFacet(facet) || 'month',
+            //     toDate = moment.utc(term.key);
+            // toDate.add(1, interval + 's');
+            // toFilter    = _.findWhere(filters, { 'field' : field + '.to',   'term' : toDate.format().slice(0,10) }),
+            // fromFilter  = _.findWhere(filters, { 'field' : field + '.from', 'term' : term.key });
```

Comment:
Is this not actually in use anywhere at the moment? Just double checking it wasn't accidentally left commented out.

Reply:
We do have "date_created" field which is meant to be date_histogram, but don't think it's currently used as facet. This feature might not have been fully implemented in which case is worth removing or commenting out, but am not 100% sure. Would probably just comment out entire `if` block though so that it does `throw new Error('Histogram not currently supported.');` (line below) if `facet.aggregation_type === 'date_histogram'`.

Reply:
Just need to comment out line 34 -> `// } else if (facet.aggregation_type === 'date_histogram'){`